src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
+    purchaseItem: '',
+    howSoon: '7',
+    lastPurchaseDate: null,
+  });
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (state.purchaseItem.trim()) {
+      let token =
+        localStorage.length !== 0 ? localStorage.getItem('token') : 'LA';
+      db.collection(token)
+        .add({
+          purchaseItem: state.purchaseItem,
+          howSoon: state.howSoon,
+          lastPurchaseDate: state.lastPurchaseDate,
+        })
+        .then(() => {
+          alert('Purchase Item has been recorded');
+        })
+        .catch((error) => {
+          alert(error.message);
+        });
+
+      setState({
+        purchaseItem: '',
+        howSoon: '7',
+        lastPurchaseDate: null,
+      });
+    } else {
+      alert('Please write an item');
+    }
+  };
+
   return (
-    <main>
-      <h1>Add Item</h1>
-    </main>
+    <>
+      <form className="form-container" onSubmit={handleSubmit}>
+        <label htmlFor="purchaseItem">Purchase Item Name:</label>
+        <input
+          type="text"
+          name="purchaseItem"
+          value={state.purchaseItem}
+          onChange={handleChange}
+        />
+
+        <label htmlFor="howSoon">
+          How soon are you likely to buy it again?:<br></br>
```

Comment:
The `<br>` tag is a self-closing tag, there's no need for a closing tag. Also, I'm not quite sure why the element is being used here.

Reply:
Thank you @C-Zekeri, when I try to remove it gives an error. I think for react is different. But I realized the grid is already doing that work of adding space. So I removed it.